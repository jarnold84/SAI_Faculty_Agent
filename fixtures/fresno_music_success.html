
<!-- Golden Fixture: Fresno State Music Faculty Directory -->
<!-- This represents a successful directory_table extraction -->
<!-- URL: https://cah.fresnostate.edu/about/directory/music/index.html -->
<!-- Expected: 36 faculty members with names, titles, profile URLs -->
<!-- Strategy: directory_table -->
<!-- Created: [Current Date] -->
<!-- DO NOT MODIFY - This is a regression test fixture -->

<!-- Note: We need to capture the actual HTML from the successful scrape -->
<!-- This is a placeholder - we'll need to manually save the real HTML -->
