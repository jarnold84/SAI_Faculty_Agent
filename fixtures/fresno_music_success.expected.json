
{
  "success": true,
  "total_found": 36,
  "strategy_used": "directory_table",
  "source_url": "https://cah.fresnostate.edu/about/directory/music/index.html",
  "items_sample": [
    {
      "name": "Earnhart, Cari",
      "title": "Department Chair",
      "email_status": "missing",
      "profile_url": "https://cah.fresnostate.edu/about/directory/music/earnhart-cari.html"
    }
  ],
  "validation_rules": {
    "min_items": 35,
    "max_items": 40,
    "required_fields": ["name", "title", "profile_url"],
    "profile_url_pattern": "https://cah.fresnostate.edu/about/directory/music/"
  }
}
